common-rendering/src/components/Pagination.tsx

```diff
@@ -23,7 +23,7 @@ type Props = {
 	format: ArticleFormat;
 };
 
-const Container = ({ children }: { children: React.ReactNode }) => (
+const NavWrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
I wanted to make sure there were no other places where the names 'Container' and 'Section' were being used so there's a bunch of renaming commits included in this PR